### PR TITLE
Updated UserAccount model to extend base user

### PR DESCRIPTION
**Pull request by mayasa3**
- Renamed 'login' app as 'accounts'

- UserAccount model now extends django default User model

- Fixed model/variable names and dependencies

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -9,9 +10,12 @@ class Gender(models.TextChoices):
...
-    user_name = models.CharField(max_length=30)
-    password = models.CharField(max_length=30)
-    gender = models.CharField(max_length=2, choices=Gender.choices, default=Gender.UNSPECIFIED, blank=True, null=True)
+    user = models.OneToOneField(User, related_name='useraccount', on_delete=models.CASCADE, blank=True, null=True)
+    gender = models.CharField(max_length=2, choices=Gender.choices, default=Gender.UNSPECIFIED)
     contact_number = models.CharField(max_length=12, blank=True, null=True)
-    user_birth_date = models.DateField()
-    user_email = models.EmailField(max_length=254, null=True)
+    birth_date = models.DateField(null=True, blank=True)
+    # future addition: email verification
```
Comment (bardielle) [inline accounts/models.py]: remove the lines in comments OR add a TODO task 
`# TODO: add email verification - issue<number> `

and remove the line 18

**Review by bardielle — CHANGES_REQUESTED** [review]
Fix the flake8 error
create a new issue for email validation and remove the comment

File: accounts/models.py (hunk 1 above)
Comment (mayasa3) [reply accounts/models.py]: Changed it, but I think I need to put that in a different PR so I removed the commit for now

**Comment by mayasa3** [comment]
Removed the comments

**Review by bardielle — APPROVED** [review]
/lgtm

**Pull request merged**